### PR TITLE
[14.0][FIX] account_asset_management: several forward ports

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Forward port of https://github.com/OCA/account-financial-tools/pull/1162.

### CODE REVIEW

**Comment by StefanRijnhart** [comment]
In the port to 14, tests were refactored. The profiles are now created in the test itself and not loaded as data.

**Comment by MiquelRForgeFlow** [comment]
It seems people are a bit lazy doing forward ports...

**Comment by HaraldPanten** [comment]
This FIX is still fighting you, @MiquelRForgeFlow 😂

**Comment by MiquelRForgeFlow** [comment]
@joao-p-marques your commit is giving me some headaches. Could you help me?

**Comment by joao-p-marques** [comment]
> @joao-p-marques your commit is giving me some headaches. Could you help me?

@MiquelRForgeFlow yes, it shouldn't introduce much complexity, looks like an access error. I've seen you tried adding the correct groups, but can you check that the analytic account and tags that are used in the test are associated to the same company as the asset and the other resources? v14 introduces some constraints there...

**Comment by MiquelRForgeFlow** [comment]
it's green :green_apple: 

@StefanRijnhart @HaraldPanten @joao-p-marques @victoralmau

File: account_asset_management/models/account_asset_profile.py [hunk 1]
```diff
@@ -177,7 +179,10 @@ def _selection_method_time(self):
         Install the 'account_asset_management_method_number_end' to enable the
         'Number' and 'End' Time Methods.
```
Comment (StefanRijnhart) [inline account_asset_management/models/account_asset_profile.py]: Please update the outdated comment.

**Review by StefanRijnhart — APPROVED** [review]

File: account_asset_management/models/account_asset_profile.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply account_asset_management/models/account_asset_profile.py]: I will just remove it. In fact, that module never existed, that's why we are adding it here.

Comment (StefanRijnhart) [reply account_asset_management/models/account_asset_profile.py]: Know thy odoo history, young man :-)
https://github.com/OCA/account-financial-tools/tree/10.0/account_asset_management_method_number_end

Comment (MiquelRForgeFlow) [reply account_asset_management/models/account_asset_profile.py]: ok, thanks haha

**Review by joao-p-marques — APPROVED** [review]
Looks good for the changes involving the port of my commit :+1:

**Pull request merged**